static/lang/re-en.json

```diff
@@ -3871,6 +3871,10 @@
                 "Tenth": "10th-Rank",
                 "Third": "3rd-Rank"
             },
+            "Spellshape": {
+                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[(@item.level)d4[fire]] damage. If the creature summoned has the @Damage[(@item.level)d4[acid]]{acid}, @Damage[(@item.level)d4[cold]]{cold}, @Damage[(@item.level)d4[electricity]]{electricity}, @Damage[(@item.level)d4[sonic]]{sonic}, or @Damage[(@item.level)d4[spirit]]{spirit} trait, you can deal damage of that type instead.",
+                "Label": "Spellshape"
```

Comment:
No need for this one, we already have a localized label for the Spellshape trait in `en.json` :)